actionpack/lib/action_dispatch/routing/mapper.rb

```diff
@@ -64,6 +64,7 @@ def self.build(scope, set, ast, controller, default_action, to, via, formatted,
 
           defaults = (scope[:defaults] || {}).dup
           scope_constraints = scope[:constraints] || {}
+          to ||= options[:to]
```

Comment:
Can we extract this `to` option where `build` is being called and pass it as the correct argument?